test/e2e/framework/syncer.go

```diff
@@ -503,6 +523,64 @@ func (sf *syncerFixture) Start(t *testing.T) *StartedSyncerFixture {
 		SyncerVirtualWorkspaceConfig:   syncerVWConfig,
 		UpsyncerVirtualWorkspaceConfig: upsyncerVWConfig,
 	}
+}
+
+// SimulateStartWithoutSyncing simulates the start of a Syncer and performs what the Syncer does to bring and maitain
+// the SyncTarget to a Ready status (heartbeat, and API schema compatibility).
+// No resource will be without effectively synced after calling this method.
+func (sf *syncerFixture) SimulateStartWithoutSyncing(t *testing.T) *StartedSyncerFixture {
+	t.Helper()
+
+	artifactDir, _, err := ScratchDirs(t)
+	if err != nil {
+		t.Errorf("failed to create temp dir for syncer artifacts: %v", err)
+	}
+
+	downstreamConfig, _, downstreamKubeClient, syncerConfig, syncerID := sf.createAndApplySyncTarget(t, false, artifactDir)
+
+	ctx, cancelFunc := context.WithCancel(context.Background())
+	t.Cleanup(cancelFunc)
+
+	kcpBootstrapClusterClient, err := kcpclientset.NewForConfig(syncerConfig.UpstreamConfig)
+	require.NoError(t, err)
+	kcpSyncTargetClient := kcpBootstrapClusterClient.Cluster(syncerConfig.SyncTargetPath)
+
+	// Artificially consider that all the resources to sync are authorized on the physical cluster side.
+	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		syncTarget, err := kcpSyncTargetClient.WorkloadV1alpha1().SyncTargets().Get(ctx, syncerConfig.SyncTargetName, metav1.GetOptions{})
+		require.NoError(t, err)
+		conditions.Set(syncTarget, conditions.TrueCondition(workloadv1alpha1.SyncerAuthorized))
```

Comment:
am curious where this is needed? Do we depend on conditions in controller or VW logic?

Reply:
We don't depend on this condition you're right, only in some tests that can then explicitely use the whole Syncer Fixture.
I removed it.